README.md

```diff
@@ -1 +1,207 @@
-# oci-kubernetes-monitoring
\ No newline at end of file
+# Monitoring Solution for Kubernetes
+
+## About
+
+This provides an end-to-end monitoring solution for Oracle Container Engine for Kubernetes (OKE) and other forms of Kubernetes Clusters, 
+using Logging Analytics, Monitoring other Oracle Cloud Infrastructure (OCI) Services.
+
+## Logs
+
+This solutions offers collection of various logs of a Kubernetes cluster, out of the box into OCI Logging Analytics and offer rich analytics on top of it. 
```

Comment:
This solutions offers collection of various logs of a Kubernetes cluster into OCI Logging Analytics and offer rich analytics on top of the collected logs.